docs-new/docs/graphql/core/guides/postgres/views.mdx

```diff
@@ -75,19 +65,17 @@ SELECT name, rating from authors_ordered_by_rating;
 
 ## Materialized views
 
-Compared to the standard view described above, [materialized
-views](https://www.postgresql.org/docs/current/rules-materializedviews.html)
+Compared to the standard view described above, [materialized views](https://www.postgresql.org/docs/current/rules-materializedviews.html)
 **do** store data physically in the database. Materialized views are
 used if data from complex queries needs to be accessed quickly.
 
 ### Example
 
-<div id="pg_materialized_view_example">
+
 
 **Materialized view with authors whose rating is larger than 6 and who
 are active, ordered by rating:**
-
-</div>
+<div id="pg-materialized-view-example" />
```

Comment:
this div needs to be moved before Line 76 (`**Materialized view with author...`) and ensure to leave an empty line between them